ui/src/__tests__/util/allstories.test.ts

```diff
@@ -0,0 +1,87 @@
+import {
+  Action,
+  allStoriesReducer,
+  INIT_STATE,
+} from "../../pages/AllStories/reducer";
+const mockTableData = [
+  {
+    ID: 1,
+    title: "Test1",
+    current_city: "City1",
+    year: 2000,
+    author_name: "TestName1",
+    author_first_name: "TestFirstName1",
+    author_last_name: "TestLastName1",
+    author_country: "Country1",
+    is_visible: false,
+    image_url: "image1",
+    video_url: "video1",
+    content: "content1",
+  },
+  {
+    ID: 2,
+    title: "Test2",
+    current_city: "City2",
+    year: 2000,
+    author_name: "TestName2",
+    author_first_name: "TestFirstName2",
+    author_last_name: "TestLastName2",
+    author_country: "Country2",
+    is_visible: true,
+    image_url: "image2",
+    video_url: "video2",
+    content: "content2",
+  },
+];
+
+describe("allstories table", () => {
+  describe("reducer", () => {
+    it("should set the correct tab value", () => {
+      const action: Action = { type: "SWITCH_TAB", id: INIT_STATE.tabValue };
+      expect(INIT_STATE.tabValue).toBe(0);
+      const newState = allStoriesReducer(INIT_STATE, action);
+      expect(newState.id !== 0).toBe(true);
+    });
```

Comment:
I'm confused by this test. Shouldn't it do something like?

```suggestion
    it("should set the correct tab value", () => {
      const newId = 3;
      const action: Action = { type: "SWITCH_TAB", id: newId };
      expect(INIT_STATE.tabValue).toBe(0);
      const newState = allStoriesReducer(INIT_STATE, action);
      expect(newState.tabValue).toBe(newId);
    });
```

Reply:
yeah you're right that test made no sense. Maybe writing these at 2am is not the way to go.